docs/source/ledger-integration-kit/index.rst

```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ledger-integration-kit:
```

Comment:
I don't think you need this - can't you just use a `:doc:` reference and refer to the page?